content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,162 @@
+
+- we need to be able to do things in English
+- we need to be able to incorporate without living there
+- we want its government to be privacy friendly and not authoritarian
+
+Your own requirements will obviously differ and some countries that we disregard in the next section might be the perfect fit for you. Or you might want to get out of the EU to not deal with [VAT MOSS](https://blog.wearewizards.io/looking-at-payments-solution-for-saas-in-europe) like [Ghost](https://blog.ghost.org/moving-to-singapore/) for example.
+
+## Countries selection
+
+France, Germany, Spain, Portugal, the Netherlands and Denmark fail as I couldn't find all the needed information on incorporation in English and the local language is required for all interactions.
+
+The USA (via [Stripe Atlas](https://stripe.com/atlas)) also doesn't make much sense from a EU citizen point of view as we have access to better choices as we will see in a moment and opens you to stupid patent laws.
+This might be different if you are planning to raise money in the US though.
+
+With the rejected countries out of the way, let's go over the potential choices I have identified after a bit of research.
+
+To keep the each country information DRY (Don't Repeat Yourself), one advice common to every country including your own is to get an accountant from the start as they are not that expensive in the grand scheme of things and let you worry only about your business.
+
+Do note that you can Google "company formation country_X" and you will get lots of companies offering services helping you with incorporation and taxes.
+
+### United Kingdom
+#### Trivia
+The UK is very business friendly and the biggest tech centre in Europe. You can probably go to a tech meetup every day if you wanted to and you get to meet and work with people from all over the world.
+
+In terms of privacy laws, the UK is not doing too great with the [IP bill](https://en.wikipedia.org/wiki/Investigatory_Powers_Bill) it's trying to pass.
+
+The obvious elephant in the room is **Brexit**. No one really knows when it will actually happen and what will be the outcome so there is a great deal of uncertainty about, well, everything.
+The biggest uncertainty for us is our residency status as we are not UK citizens and there is a greater than zero risk that we would not be able to stay. Getting citizenship is not an option for me for example as I'm moving out of the country soon.
+
+Sadly enough, EU VAT handling could [become even harder than the current VAT MOSS for UK businesses](http://www.bmmagazine.co.uk/news/uk-leave-eu-vat-regime-following-brexit/).
+
+The UK is currently the place to be in Europe if you want to get investment but it might change after Brexit.
+
+#### Taxes
+The corporation tax is currently **20%** but there are [talks to reduce it 17% by 2020](http://www.telegraph.co.uk/business/2016/07/04/will-osbornes-plan-to-cut-corporation-tax-help-boost-the-post-br/) and potentially even further.
```

Comment:
fixed the typo.
That's why the article is linked, they want to reduce it faster to attract companies despite brexit